locale/fr_FR/locale.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<locale name="fr_FR" full_name="Français (France)">
+
+	<!-- Theme description -->
+	<message key="plugins.themes.immersion.name">Theme Immersion</message>
+	<message key="plugins.themes.immersion.description">Un thème pour OJS 3.1.1+ qui met l'accent sur une expérience de lecture immersive et offre des options de conception audacieuses telles que des choix de couleurs dans toute la section.</message>
+
+	<!-- Forms -->
+	<message key="plugins.themes.immersion.colorPick">Selecteur de couleur</message>
+	<message key="plugins.themes.immersion.colorPickInstructions">Choisissez une couleur pour une section à afficher sur le front-end. Doit être un code hexadécimal HTML valide (par exemple, # ffff00 pour le jaune).</message>
+	<message key="plugins.themes.immersion.section.coverPage">Image de couverture de la section</message>
+	<message key="plugins.themes.immersion.announcements.colorPick">Couleur de la section annonce de la page d'accueil</message>
+	<message key="editor.issues.sectionArea">Options pour les sections des numéros</message>
+
+	<!-- General front-end -->
+	<message key="plugins.themes.immersion.issue.published">Publié</message>
+	<message key="plugins.themes.immersion.issue.description">Description du numéro</message>
+	<message key="plugins.themes.immersion.registration.consent">Consentement</message>
+	<message key="plugins.themes.immersion.mainMenu">Menu principal</message>
+	<message key="plugins.themes.immersion.language.toggle">Changer de langue. La langue actuelle est:</message>
+	<message key="plugins.themes.immersion.adminMenu">Administration</message>
+	<message key="plugins.themes.immersion.about.journal">À propos du journal journal</message>
```

Comment:
Thanks. Can you review this one? It should be just  `À propos du journal`, right?

Reply:
Hello, review done !